### DISCUSSION TITLE
[Phaser] empty containers

### DISCUSSION THREAD

**Discussion by E-A-N** [enable3d/enable3d#173] [Help / Questions] [answered]
How do I create an empty container to store objects in without using THREE?  I'm ultimately looking to create a searchable grid.

**Comment by yandeu** [comment]
You mean something like [Map()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) or [Set()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set)?

**Reply by E-A-N** [reply]
I'm sorry when I say container I'm referring to group.  What's the enabled3D equivalent to: 

```javascript
//threejs
const cubeA = new THREE.Mesh( geometry, material );
const group = new THREE.Group();
group.add( cubeA );
```

I want to create grid of cubes and put them all in a group.

**Reply by yandeu** [reply]
Why don't you want to use THREE.Group()? This should work.

**Comment by E-A-N** [comment, marked as answer]
So [this](https://enable3d.io/examples/shape-showcase.html) example contains a rough equivalent of what I'm looking for.  I guess I have to make my own group essentially?

```javascript
create() {
          this.third.warpSpeed('light', 'orbitControls', 'sky')

          this.objects = [
            this.third.add.box({ x: -4, y: 0, z: 0 }),
            this.third.add.sphere({ x: -2, y: 0, z: 0, radius: 0.5 }),
            this.third.add.sphere({
              x: 0,
              y: 0,
              z: 0,
              radius: 0.5,
              widthSegments: 6,
              heightSegments: 4
            }),
            this.third.add.sphere({ x: 2, y: 0, z: 0, radius: 0.5, phiLength: Math.PI }, { lambert: { side: 2 } }),
            this.third.add.cylinder({
              x: 4,
              y: 0,
              z: 0,
              radiusTop: 0.5,
              radiusBottom: 0.5,
              height: 1
            })
          ]
        }

        update() {
          this.objects.forEach(obj => {
            obj.rotation.x += 0.01
            obj.rotation.y += 0.01
          })
        }
      }
      ```

**Reply by yandeu** [reply]
This is a simple array. But you could also write your own simple Group class with `group.add()` and `group.searchById()` for example.
